sierra_adapter/sierra_item_merger/src/test/scala/uk/ac/wellcome/platform/sierra_item_merger/locals/DynamoDBLocal.scala

```diff
@@ -0,0 +1,63 @@
+package uk.ac.wellcome.platform.sierra_item_merger.locals
```

Comment:
This could live in the common lib and be shared with the bib merger, surely? They have the same table structure, yes?